[2025-02-23T15:42:04.038+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-23T15:42:04.071+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bangalore_weather_pipeline.transform_weather_data scheduled__2025-02-23T14:00:00+00:00 [queued]>
[2025-02-23T15:42:04.082+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bangalore_weather_pipeline.transform_weather_data scheduled__2025-02-23T14:00:00+00:00 [queued]>
[2025-02-23T15:42:04.082+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-23T15:42:04.101+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): transform_weather_data> on 2025-02-23 14:00:00+00:00
[2025-02-23T15:42:04.106+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=20816) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-23T15:42:04.108+0000] {standard_task_runner.py:72} INFO - Started process 20817 to run task
[2025-02-23T15:42:04.108+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'bangalore_weather_pipeline', 'transform_weather_data', 'scheduled__2025-02-23T14:00:00+00:00', '--job-id', '145', '--raw', '--subdir', 'DAGS_FOLDER/bangalore_weather_pipeline/bangalore_weather_etl.py', '--cfg-path', '/tmp/tmp7jd918h1']
[2025-02-23T15:42:04.110+0000] {standard_task_runner.py:105} INFO - Job 145: Subtask transform_weather_data
[2025-02-23T15:42:04.187+0000] {task_command.py:467} INFO - Running <TaskInstance: bangalore_weather_pipeline.transform_weather_data scheduled__2025-02-23T14:00:00+00:00 [running]> on host ee2081ae3a04
[2025-02-23T15:42:04.320+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='bangalore_weather_pipeline' AIRFLOW_CTX_TASK_ID='transform_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-23T14:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-23T14:00:00+00:00'
[2025-02-23T15:42:04.321+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-23T15:42:04.324+0000] {python.py:240} INFO - Done. Returned value was: {'latitude': '12.9716', 'longitude': '77.5946', 'temperature': 23.9, 'windspeed': 5.9, 'winddirection': 76, 'weathercode': 0}
[2025-02-23T15:42:04.365+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-23T15:42:04.365+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=bangalore_weather_pipeline, task_id=transform_weather_data, run_id=scheduled__2025-02-23T14:00:00+00:00, execution_date=20250223T140000, start_date=20250223T154204, end_date=20250223T154204
[2025-02-23T15:42:04.442+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-23T15:42:04.494+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-23T15:42:04.497+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
